diff --git a/dist/changesets-cli.cjs.js b/dist/changesets-cli.cjs.js
index 0f03e31068a5a7f77ef03ee97684d7f4d5136367..d7007c613c70dd51a8b73e584fb413054b431f20 100644
--- a/dist/changesets-cli.cjs.js
+++ b/dist/changesets-cli.cjs.js
@@ -762,7 +762,9 @@ async function internalPublish(packageJson, opts, twoFactorState) {
   // Due to a super annoying issue in yarn, we have to manually override this env variable
   // See: https://github.com/yarnpkg/yarn/issues/2935#issuecomment-355292633
   const envOverride = {
-    [scope ? `npm_config_${scope}:registry` : "npm_config_registry"]: registry
+    [scope ? `npm_config_${scope}:registry` : "npm_config_registry"]: registry,
+    // Don't pass through any npm auth tokens so that we use trusted publishers
+    NODE_AUTH_TOKEN: ""
   };
   let {
     code,
diff --git a/dist/changesets-cli.esm.js b/dist/changesets-cli.esm.js
index 1946941b0b876b6f90d25c613696a8e151b726c4..c021d21c9054f6c12ef9d3ba869cc9e4b8217f29 100644
--- a/dist/changesets-cli.esm.js
+++ b/dist/changesets-cli.esm.js
@@ -724,7 +724,9 @@ async function internalPublish(packageJson, opts, twoFactorState) {
   // Due to a super annoying issue in yarn, we have to manually override this env variable
   // See: https://github.com/yarnpkg/yarn/issues/2935#issuecomment-355292633
   const envOverride = {
-    [scope ? `npm_config_${scope}:registry` : "npm_config_registry"]: registry
+    [scope ? `npm_config_${scope}:registry` : "npm_config_registry"]: registry,
+    // Don't pass through any npm auth tokens so that we use trusted publishers
+    NODE_AUTH_TOKEN: ""
   };
   let {
     code,
